EnglishFilter/sections/specific.txt

```diff
@@ -4904,7 +4904,7 @@ hdzog.com##.main-content > .block-thumbs:not([class*=" "])
 4tube.com###ad_player
 ||shaggyimg.pro/crackle/carouseal.js
 gamefaqs.gamespot.com##.message_mpu
-|https://$script,subdocument,third-party,xmlhttprequest,domain=upload4earn.org
+|https://$script,subdocument,third-party,xmlhttprequest,domain=upload4earn.org|moviesflix.one
```

Comment:
mirror: themoviesflix.co
better add to regex rule 
https://github.com/AdguardTeam/AdguardFilters/commit/730a983e7b86598456b8fdbe28ee3817d50920a2#diff-8e737f6429801c4aac12f0a8ddb7b4ab3173f39642bfe56004e22dede37edd6dR217

Reply:
@Alex-302 It doesn't do anything except blocking one ad-server. It doesn't block the others.

Reply:
Okay.

Reply:
Missed the mirror - themoviesflix.co 
Added in my commit